__tests__/api.test.js

```diff
@@ -31,3 +32,60 @@ describe('GET/api/topics', () => {
         });
     });
 });
+
+
+describe('GET/api', () => {
+    it('200: responds with all endpoints available', () => {
+        return request(app).get('/api').expect(200);
+    })   
+
+    it('200: responds with the expected JSON describing all available endpoints', () => {
+        return request(app).get('/api')
+        .then((response) => {
+        expect(response.body['GET /api'].description).toBe('serves up a json representation of all the available endpoints of the api');
+        expect(response.body['GET /api/topics'].description).toBe('serves an array of all topics');
+        expect(response.body['GET /api/articles'].description).toBe('serves an array of all articles');
+        })   
+    });
+});
+
+describe('GET/api/articles/:article_id', () => {
+    it('200: responds with an article object', () => {
```

Comment:
Really good first test, I'd maybe adjust the test description to say just "responds with status 200" or something along those lines, because in this test it's only checking you get the `200` code